hostinfo/hostinfo.go

```diff
@@ -28,6 +28,7 @@ const (
 	Heroku          = EnvType("hr")
 	AzureAppService = EnvType("az")
 	AWSFargate      = EnvType("fg")
+	FlyDotIo        = EnvType("fy")
```

Comment:
Not sure how we got on this two letter code thing, but sure?

I'd also accept `fly` here :)

Reply:
Went with `fly`